sonarts-core/src/rules/noUnenclosedMultilineBlockRule.ts

```diff
@@ -114,15 +114,14 @@ class Visitor extends SonarRuleVisitor {
   }
 
   private countStatementsInTheSamePile(reference: ts.Statement, statements: ts.Statement[]): number {
-    const file = reference.getSourceFile();
-    let startOfPile = lineAndCharacter(reference.getStart(), file);
+    let startOfPile = startLineAndCharacter(reference);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Identifier 'startOfPile' is never reassigned; use 'const' instead of 'let'. ([typescript:S3353](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS3353))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=sonarts&branch=issue-489&issues=AWGz_GjGUhrvXJpox_le&open=AWGz_GjGUhrvXJpox_le)

[ISSUE_KEY]: AWGz_GjGUhrvXJpox_le